src/services/identity/DeploymentsService.ts

```diff
@@ -152,17 +154,38 @@ class DeploymentsService {
         )
 
         // 3. Create branches
-        if (createStagingLite) {
+        if (isStagingLite) {
           return this.deploymentClient
             .sendCreateBranch(createStagingLiteBranchInput)
+            .andThen(() =>
+              this.deploymentClient.sendStartJobCommand({
+                appId: amplifyInfo.id,
+                branchName: "staging-lite",
+                jobType: "RELEASE",
+              })
+            )
             .map(() => amplifyInfo)
         }
         return this.deploymentClient
           .sendCreateBranch(createMasterBranchInput)
+          .andThen(() =>
+            this.deploymentClient.sendStartJobCommand({
+              appId: amplifyInfo.id,
+              branchName: "master",
+              jobType: "RELEASE",
+            })
+          )
           .map(() => amplifyInfo)
           .andThen(() =>
             this.deploymentClient
               .sendCreateBranch(createStagingBranchInput)
+              .andThen(() =>
+                this.deploymentClient.sendStartJobCommand({
```

Comment:
how is it working currently without this btw?

Reply:
I am not sure why this doesnt work all the time